src/scipp/core/binning.py

```diff
@@ -191,10 +191,10 @@ def hist(x: Union[_cpp.Variable, _cpp.DataArray, _cpp.Dataset],
 
     1. When an integer is provided, a 'linspace' with this requested number of
        bins is created, based on the min and max of the corresponding coordinate.
-    2. A scalar scipp variable (a value with a unit) is interpreted as a target
+    2. A scalar Scipp variable (a value with a unit) is interpreted as a target
```

Comment:
Should we really change these? It conceptually refers to `scipp.Variable`, so uppercasing amy be confusing?

Reply:
Then 'variable' should be uppercase. I would say we either use code-notation or natural language.